CHANGELOG.md

```diff
@@ -15,17 +15,25 @@ it according to semantic versioning. For example, if your PR adds a breaking cha
 should change the heading of the (upcoming) version to include a major version bump.
 
 -->
+
 # 5.14.3
 
+## @rjsf/utils
+
+- Added an experimental flag `allOf` to `experimental_defaultFormStateBehavior` for populating defaults when using `allOf` schemas [#3969](https://github.com/rjsf-team/react-jsonschema-form/pull/3969)
+
```

Comment:
Can you move this after `@rjsf/fluentui-rc` since we like to keep the packages in alphabetical order. And there are also conflicts due to another merged PR. Finally, it seems like you did a major reformat of the file (your IDE?). Can you revert all of those changes to previous versions?

Reply:
Woops, yep I thought I had saved without formatting for that file. Updated and made the changes :tadA:

Reply:
@heath-freenome not sure about the build failures here, seemed to have rebased but they look unrelated?

Reply:
@zxbodya Could this be related to your recent PR?

Reply:
it indeed looks to be because of project references misconfiguration -- added fix here #3982